src/suresh/arrayExample4.java

```diff
@@ -0,0 +1,76 @@
+package suresh;
+
+public class arrayExample4 {
+
+	int listNumbersAsc() {
+		int[] num = new int[5];
+		for (int index = 0; index < num.length; index++) {
+			num[index] = index + 1;
+			System.out.println("Series of numbers:" + index);
+		}
+		return 0;
+	}
+
+	int listNumbersDesc() {
+		int[] num = new int[0];
+		for (int index = 5; index > num.length; index--) {
+			System.out.println("Length to 0:" + index);
+		}
+		return 0;
+	}
+
+	int maxNumber(int num[]) {
+		int max = num[0];
+
+		for (int index = 0; index < num.length; index++) {
+
+			if (max < num[index])
+				max = num[index];
+		}
+		return max;
+	}
+
+	int minNumber(int[] input) {
+		int min = 0;
+		for (int index = 0; index < input.length; index++) {
+			if (input[index] < min)
+				min = input[index];
+		}
+		return min;
+	}
+
+	int avgNumber(int[] input) {
+		int total = 0;
+		for (int index = 0; index < input.length; index++) {
+			total = total + input[index];
+		}
+		double average = total / input.length;
+		System.out.println("The average is: %.3f" +average);
+		return total;
+	}
+
+	int avgMinMax(int[] input) {
+		
+		int max = maxNumber(input);
+		int min = minNumber(input);
+		
+		int avg = (min+max)/2;
+		
+		return avg;
```

Comment:
You can write code like this, I remove variable and directly doing sum of min & max.
no problem, if you are not comfortable to write code like below, just FYI.

int avgOfMinMax = (min(input)+max(input))/2